kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -409,14 +389,14 @@ fn collect_codegen_items<'tcx>(gcx: &GotocCtx<'tcx>) -> Vec<MonoItem<'tcx>> {
     }
 }
 
-fn symbol_table_to_gotoc(tcx: &TyCtxt, file: &Path) {
-    let output_filename = file.with_extension("symtab.out");
-    let input_filename = file.with_extension("symtab.json");
+fn symbol_table_to_gotoc(tcx: &TyCtxt, file: &Path) -> impl AsRef<Path> {
```

Comment:
that seems like an especially complicated return type

Reply:
I'll update it